yellow-paper/docs/public-vm/gen/_InstructionSet.mdx

```diff
@@ -1202,23 +1221,6 @@ Get the historical l1-to-l2 message tree root as of the specified block number.
 
 [![](./images/bit-formats/MSGSROOT.png)](./images/bit-formats/MSGSROOT.png)
 
-### <a id='isa-section-notesroot'/>`NOTESROOT` (0x28)
```

Comment:
I honestly am still not sure what `ROOT` or "tree read" operations we actually need....

Reply:
Neither tbh, but i removed this as it was a duplicate with line 743!! Not a blanket removal

https://github.com/AztecProtocol/aztec-packages/blob/91025fa7265e7e80992ae7bc2ea0e38a85f009df/yellow-paper/src/preprocess/InstructionSet/InstructionSet.js#L765